uportal-war/src/main/webapp/media/skins/respondr/common/less/navigation.less

```diff
@@ -43,6 +45,47 @@
         }
     }
 
+   .logout-launcher,.notification-icon,.portal-greeting,.login-launcher{
+        display: block;
+        float: right;
+        margin-right: 5px;
+        margin-top: 12px;
+        word-break: keep-all;
+    }
+
+    #portalCASLogin{
+        display: block;
+        float: right;
+        margin-right: 5px;
+        margin-top: -5px;
+    }
```

Comment:
I think #portalCASLogin is specific to your implementation and not needed. If meant for contribution, should probably be made general such as #portalLogin.